src/utils/digitalPlanningSchema.ts

```diff
@@ -19,7 +19,7 @@ export function getValidSchemaValues(definition: string): string[] | undefined {
 }
 
 /**
- * For a given 'anyOf' enum definition in the JSON Schema (with properties `value` & `description`), return a dictionary of its valid options
+ * For a given 'anyOf' enum definition in the Application JSON Schema (with properties `value` & `description`), return a dictionary of its valid options
```

Comment:
These helper functions (which we call in planx-new for things like publish validation checks) are going to continue only reading against the `application` schema - but I think this is generally fine & expected behavior! Enums referenced throughout `preApplication` are shared anyways currently.